_content/overview.md

```diff
@@ -1,100 +1,103 @@
-- Current pop-culture references.
-- Jokes.
-- Exclamation marks, except in code examples.
-- Wackiness, zaniness, and goofiness.
-- Phrasing that denigrates, insults, or discriminates against any group of people.
-- Using phrases like "simply" or "it is that simple" or "it is easy" in a procedure.
-- Internet slang, or other [internet abbreviations](https://developers.google.com/style/abbreviations#dont-use) such as "[tl;dr](https://developers.google.com/style/word-list#tldr)" or "[ymmv](https://developers.google.com/style/word-list#ymmv)".
-- Do not use dangling modifiers. A modifier "dangles" when the sentence is not clear about what is being modified.
+- Buzzwords, jargon, or slang.
+- Humorous or absurdist references.
+- References to pop culture or the names of real people.
+- [Ableist](https://developers.google.com/style/inclusive-documentation.html#ableist-language), gendered, or discriminatory language or figures of speech, such as "crazy," "crippled," or "mankind." When necessary, use the singular pronoun "they" instead of "he" or "she."
+- Empty phrases, such as "please note," "for the most part," or "at this time."
+- Convoluted or overly long sentences.
+- Starting all sentences with the same phrase, such as "You can" or "To do."
+- Exclamation marks (except in code examples).
+- Judgment phrases, such as "simply" or "it is that simple" or "it is easy" in a procedure.
+- Dangling modifiers. A modifier "dangles" when the sentence is not clear about what is being modified.
+- Double negatives 
 
 #### Accessible content
 
-Our audience is comprised of people who come to our website with different abilities. Some might have issues with their eyesight and magnify the content on their screen. Others may use a screen reader that speaks the text on a web page. Some people come to the Agora website for whom English is a second language, and their reading comprehension level may be low. Content aimed at U.S. readers should be in line with the Americans with Disabilities Act (ADA) and use the [W3C Web Content Accessibility Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/) as a guide. By following the best practices outlined previously, you will already be doing most of what is required, but also keep the following in mind when you write:
+Our readers come to our website with different abilities. Some may have issues with their eyesight and magnify the content on their screen. Others may use a screen reader that speaks the text on a web page. For some, English is their second language, and their reading comprehension level may be low. Content aimed at U.S. readers should be in line with the Americans with Disabilities Act (ADA) and use the [W3C Web Content Accessibility Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/) as a guide. By following the best practices outlined previously, you will already be doing most of what is required, but also keep the following in mind when you write:
```

Comment:
We have discussed about the usage of "may" and "might", and decided to follow Google's style guide and use might to refer to possibilities.

Reply:
Agreed. I will fix this.